drivers/amazonec2/amazonec2.go

```diff
@@ -903,15 +889,13 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) []*
 	hasDockerPort := false
 	hasSwarmPort := false
 	for _, p := range group.IpPermissions {
-		if p.FromPort != nil {
```

Comment:
Hm, looks like there's a few issues with rebase, so I'm guessing this wasn't intentional.